src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const tableNavigation = table.querySelector('thead');
+const tableBody = table.querySelector('tbody');
+const tableBodyRows = Array.from(table.querySelector('tbody').rows);
+
+const sortTable = (index, sortMethod = 1) => {
+  tableBodyRows.sort((a, b) => {
+    let partA = a.children[index].innerHTML;
+    let partB = b.children[index].innerHTML;
+
+    if (sortMethod) {
+      partA = partA.replace(/\D/g, '');
+      partB = partB.replace(/\D/g, '');
+
+      return partA - partB;
+    } else {
+      return partA.localeCompare(partB);
+    }
+  }).map(el => tableBody.append(el));
```

Comment:
```suggestion
  }).forEach(el => tableBody.append(el));
```
Не дуже коректно використовувати тут метод map. Метод map повертає нам новий масив із зміненими едементами. Тут же тобі не потрібно повертати новий масив, а просто викоати певні операції з кожним з елементів поточного масиву

Reply:
оке, а как мне сделать универсально функцию ?
У нас есть столбики где сортировка идет по буквам, другие - по цифрам

если бы было много разных страниц на сайте

Reply:
```const methodSort = index <= 1 ? 0 : 1;```

Таким образом, я определял 
какая будет сортировка -- по числам или же по буквам 

А если будет таблица другая, этот уже не прокатит